cli/Cargo.toml

```diff
@@ -16,5 +16,5 @@ path = "src/bin/hwi.rs"
 clap = { version = "4.4.7", features = ["derive"] }
 bitcoin = "0.31"
 hex = "0.4"
-async-hwi = { version = "0.0.17" }
+async-hwi = { path = "../" }
```

Comment:
Using path makes the crate unplishable on crates.io but I learned that we can have both:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations
> It is possible to specify both a registry version and a git or path location. The git or path dependency will be used locally (in which case the version is checked against the local copy), and when published to a registry like [crates.io](https://crates.io/), it will use the registry version

Can you change it to have both ?